content/applications/marketing/email_marketing/lost_leads_email.rst

```diff
@@ -0,0 +1,337 @@
+
+
+   .. image:: lost_leads_email/stages.png
+      :align: center
+      :alt: Three filter rules requiring that the Stage does not contain New, Qualified, or Won.
+
+Lost Reason
+-----------
+
+Next, add one or more :guilabel:`Lost Reason` rules to exclude leads that should not be targeted for
+specific :doc:`lost reasons <../../sales/crm/pipeline/lost_opportunities>`.
+
+Create another :guilabel:`New Rule`, and in the rule's first field, select :guilabel:`Lost Reason`
+from the drop-down menu. For the operator, choose either :guilabel:`is not in` or :guilabel:`does
+not contain` from the drop-down menu. With either selection, use the third field to enter a lost
+reason (or multiple lost reasons, depending on your operator choice) to include in the rule.
+
+If choosing the :guilabel:`does not contain` operator, then repeat the preceding steps to add more
+lost reasons, as needed, where each lost reason occupies one rule row at a time.
+
+For more information, refer to the section below outlining how to :ref:`select appropriate lost
+reasons <email_marketing/select_lost_reasons>`.
+
+.. image:: lost_leads_email/reasons.png
+   :align: center
+   :alt: A list of filter rules that exclude all lost reasons other than the desired reason.
+
+Active
+------
+
+Finally, add a pair of :guilabel:`Active` filters to include both active and inactive leads for the
+campaign.
+
+.. important::
+   Adding both active **and** inactive lead records is necessary to capture the full scope of lost
+   leads in the database. Doing one without the other greatly impacts the number of targetable
+   records for the email campaign, and will not include a complete or accurate lost leads audience.
+
+Click the :guilabel:`(Add Branch)` icon to the right of the latest rule's row to add a pair of
+:guilabel:`any of` rules. In the top rule's first field, select the :guilabel:`Active` parameter
+from the drop-down menu. The rule will then automatically fill out to read: :guilabel:`Active` *is*
+`set`.
+
+For the first field of the bottom rule, once more, select :guilabel:`Active` from the drop-down
+menu and, this time, select :guilabel:`is not` from the operator drop-down menu, so the rule reads:
+:guilabel:`Active` *is not* `set`.
+
+.. image:: lost_leads_email/active.png
+   :align: center
+   :alt: A pair of Match Any Of filter rules that include both active and inactive leads.
+
+Add body content
+================
+
+Now, with the domain section of the email campaign complete, create the body content of the email
+using any of the premade stylized templates, or choose between the :guilabel:`Plain Text` or
+:guilabel:`Start From Scratch` options for more granular control. For more information, refer to the
+*Email Marketing* :ref:`documentation on how to create an email <email_marketing/create_email>`.
+
+.. tip::
+   To save the set of filters for later use, click :guilabel:`Save as Favorite Filter 💾`, enter a
+   name (such as `Lost Leads`), and click :guilabel:`Add`.
```

Comment:
```suggestion
   To save the set of filters for later use, click :guilabel:`Save as Favorite Filter 💾 (floppy
   disk)` icon, enter a name (such as `Lost Leads`), and click :guilabel:`Add`.
```

Reply:
I'm glad you pointed this out, because I thought about adding the accessibility tag here like usual, (like we would in such a case as "Click the :guilabel:`➡️ (right arrow)` icon...) however, I decided _not_ to add it since it's not necessary for the reader's comprehension or for valuable action to take place. If the emoji doesn't render in the html, it doesn't matter.

That being said, I'm not so sure breaking the precedent is worth it, so I'll add it back in.